### DISCUSSION TITLE
Umami cloud script blocked by Brave & uBlock origin

### DISCUSSION THREAD

**Discussion by ssl-wiki-sid** [umami-software/umami#2197] [Q&A]
## Brave blocking
![image](https://github.com/umami-software/umami/assets/84215848/46704b38-422e-4edc-94ea-2d497187b00a)

## uBlock origin blocking
![image](https://github.com/umami-software/umami/assets/84215848/bdb93abc-5dea-4870-b6fe-5c8b8e1fc23f)

## Solutions which will not work

- I don't think there is an option to set environment variable in Umami cloud (beta). 
- Even if I host the JS script in my domain, I guess the network request to `https://analytics.umami.is/api/send` will be blocked.

Any idea how to overcome these limitations?

Thanks in advance.

**Comment by uncenter** [comment]
I know some blockers just block URLs with "analytics" or "umami" in them. I self-just mine which means I can just change the subdomain... maybe Umami Cloud could use a different subdomain? Something random like `bob.umami.is`?
